pkg/cloudprovider/provider/kubevirt/provider_test.go

```diff
@@ -192,7 +192,7 @@ func (k kubevirtProviderSpecConf) rawProviderSpec(t *testing.T) []byte {
 
 var (
 	userdata      = "fake-userdata"
-	testNamespace = "test-namespace"
+	testNamespace = "tenant-cluster"
```

Comment:
I find this name somehow confusing.
Maybe more `cluster-xyz` that would be more close to the real names ?